operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md

```diff
@@ -119,89 +119,37 @@ documentation (`S-8031`) for instructions on how to acquire a SAT authentication
+   Retrying container stop procedure on ncn-m002
+   WARNING: Some containers are still running after stop attempt on ncn-w001: ['21570acf6af066532bf80b2ece10c6808506f9672a03d24fd4f7e5a7775512bf', '3c346c2f0788fc68efe172f48faf9972d296713c50a709aed8b64a39ef722916', '5aebf6f06341327bbec581543e9812a20faac977fe45d870dffabf4d6f81a6c8', 'a9729fc0b2151a876ca0873562924005f8b323e4b32b7542af02a04532c8973b', 'd08a21f887a8cf6eade77459cc78cce35d4a34a25a7d80e59e8bd1cff9243ea1', 'fe5e5ae6880069042e8e2ab88a879cc68d1e934bb2a67f05504f4900aaf5461e', '4ab27e2dba2e677f3a8f9a41bc47bbdbb0059d4cef37fabfe58807147c47c68f', 'd1970d162b2f2e8f460fdba554f4aa5193c7450aa1dd0230272e18d3f6360177']
+   Retrying container stop procedure on ncn-w001
+   WARNING: Some containers are still running after stop attempt on ncn-w003: ['f8b6132ea7cbd8c222b2fbc2f98842e3acd2a69b43221dff860357edc22665ba', '542e44dddf7b0d60568651ef5f6632376b9220b67aa0df47cb2d6465a6c4b309', 'ca6d2b31c5871fe7da69e6c8ae93ca136fff9c788798b6dbd2ac6c54eb929a1c', '2e4d5bcd4c6fae9cbc3ce639561bce2aeb89c435fbe67a8721f1c1423c4cde8e', 'b81a9b6f82188a09794ad78c12b7b6ff59926d930b56e3cfd4b8d0fae595087e', 'e0c415d1a56cf567443dfd0a192077b91f38791ff51b7ce0827935bd67509238', '6a286158c0070ebc1b2ae7cc1d78fc68fa2b146980aea3aa93bffc4a20e13177', '1577cf4a6c5dfb0d4589087210b7b5f18ef3ab45785920a358f5c42a5f02f6ec']
+   Retrying container stop procedure on ncn-w003
+   All containers stopped on ncn-m002.
+   WARNING: Some containers are still running after stop attempt on ncn-w002: ['22c469499a766daeaacc40254d72f10462ae7dd7de3da7ffa25b8243f64a3779', 'ae5517e5e6de615773ccf65f81f9090eab93cc4149001149f7dd626b743b5ee2', '05d82334cef63b0260e7c17d8bd270344ff36b7c38858fc43095b8ec5ac3c38b', '727804ee10ab6ca0bbeb9be95f3ff6e60729f4bea9fefc6f1ffc2b51bac809a1', 'f4008d3e8bae37ca1bc6ed7ae24475a25f7af28d7e057337832f68a7d0f9eb03', '03a3492c20d6ba4833ebb5f625403046e6fd2d79d28efe77b33ed9cfc1ad5101', 'd0312947c43882081af564658568a1384f9bfa38b956403b4af16dc49bb225f4', 'ef9a880958778965b9f830d6cc9531c86bf8382456578933c0e1248a355e7e3b']
+   Retrying container stop procedure on ncn-w002
+   WARNING: Some containers are still running after stop attempt on ncn-w001: ['5aebf6f06341327bbec581543e9812a20faac977fe45d870dffabf4d6f81a6c8']
+   Retrying container stop procedure on ncn-w001
+   WARNING: Some containers are still running after stop attempt on ncn-w003: ['ca6d2b31c5871fe7da69e6c8ae93ca136fff9c788798b6dbd2ac6c54eb929a1c', 'e0c415d1a56cf567443dfd0a192077b91f38791ff51b7ce0827935bd67509238']
+   Retrying container stop procedure on ncn-w003
+   All containers stopped on ncn-w001.
+   WARNING: Some containers are still running after stop attempt on ncn-w002: ['05d82334cef63b0260e7c17d8bd270344ff36b7c38858fc43095b8ec5ac3c38b', '03a3492c20d6ba4833ebb5f625403046e6fd2d79d28efe77b33ed9cfc1ad5101']
+   Retrying container stop procedure on ncn-w002
+   All containers stopped on ncn-w003.
+   All containers stopped on ncn-w002.
```

Comment:
We could probably shorten the output here (and remove ncn-w004) to show just enough of the warnings to be illustrative, e.g.:

```suggestion
   All containers stopped on ncn-m001.
   All containers stopped on ncn-m003.
   All containers stopped on ncn-w003.
   All containers stopped on ncn-w002.
   WARNING: Some containers are still running after stop attempt on ncn-m002: ['f8a4d0ffe74588fcd4a6ab644cac62cc271df7681cea74173f28d66b5391873a']
   Retrying container stop procedure on ncn-m002
   WARNING: Some containers are still running after stop attempt on ncn-w001: ['21570acf6af066532bf80b2ece10c6808506f9672a03d24fd4f7e5a7775512bf', '5aebf6f06341327bbec581543e9812a20faac977fe45d870dffabf4d6f81a6c8', 'd1970d162b2f2e8f460fdba554f4aa5193c7450aa1dd0230272e18d3f6360177']
   Retrying container stop procedure on ncn-w001
   All containers stopped on ncn-m002.
   WARNING: Some containers are still running after stop attempt on ncn-w001: ['5aebf6f06341327bbec581543e9812a20faac977fe45d870dffabf4d6f81a6c8']
   Retrying container stop procedure on ncn-w001
   All containers stopped on ncn-w001.
```

Reply:
Having the example is great, but I would also suggest just a short mention of it in the text that introduces the example. Something like this:

```
   The following example output shows warnings that may occur while stopping containers on
   Kubernetes nodes. When these warnings occur, the `sat bootsys` command will continue attempting
   to stop containers until all containers are stopped.
```

Reply:
Addressed